integration_tests/suite/test_calls_stasis.py

```diff
@@ -3,20 +3,22 @@
 
 import json
 
-from hamcrest import assert_that
+from hamcrest import assert_that, has_item
 from hamcrest import has_entries
 from hamcrest import has_entry
 from hamcrest import has_items
```

Comment:
I think the imports could be cleaned up a bit like this:
```suggestion
from hamcrest import (
    assert_that,
    has_entries,
    has_entry,
    has_item,
    has_items,
)
```
You could do the same for the other imports too